### ISSUE TITLE
Not Working properly

### ISSUE THREAD

**Issue by TheDjRider** [kylgus/SFMT#6]
Hello,
I have a problem with script. It won't create _nowplaying.txt_ file. 
![image](https://user-images.githubusercontent.com/12405637/47622636-d20f2380-db0f-11e8-81f2-7bc122db90d3.png)

Thanks :)

**Comment by TheL1ne** [comment]
maybe create the file yourself?

**Comment by TheDjRider** [comment]
I create it but it won't type in the file. Try with Spotify, YouTube, VLC, windows media player and groove. 
I don't have any idea 😢

I use the last version of AutoHotkey, is that a problem ?

**Comment by MonsterPoney** [comment]
Did you change the windowTitle variable value ?
At this state, the program search for a window name who contains "Winamp"

**Comment by TheL1ne** [comment]
yeah the screenshot above states that it is looking for Winamp. This script needs to be configured. So you would need to set the windowtitle according to the title of the program you are using. Additionally this script can only extract song titles from programs that display the title in the window title as it just grabs a part of this. ( i think spotify does not do this)
